Cargo.toml

```diff
@@ -26,6 +26,7 @@ python = ["interfaces", "pyo3"]
 wasm_bindgen = ["interfaces", "js-sys", "wasm-bindgen"]
 
 [dependencies]
+abe_policy = { path = "../abe_policy" }
```

Comment:
It is a relative depedency, out of `abe_gpsw` project, so this cannot work.

You need to publish this dependency, Cargo prevents to publish any crate where any dependency is not public

Reply:
I know that :D I'm waiting someone reviews `abe_policy` before pushing it. I will update the deps afterwards.